src/app/containers/ATIAnalytics/index.jsx

```diff
@@ -24,6 +25,7 @@ const ATIAnalytics = ({ data }) => {
     mostWatched: buildMostWatchedATIUrl,
     IDX: buildIndexPageATIUrl,
     FIX: buildIndexPageATIUrl,
+    TOPIC: () => buildTopicPageATIUrl(data, requestContext, serviceContext),
```

Comment:
Line 59 is passing these arguments anyway

```suggestion
    TOPIC: buildTopicPageATIUrl,
```